java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -1070,8 +1070,12 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_binaryOpVV(JNIEnv *env, j
 
     cudf::data_type n_data_type = cudf::jni::make_data_type(out_dtype, scale);
     cudf::binary_operator op = static_cast<cudf::binary_operator>(int_op);
+<<<<<<< HEAD
```

Comment:
Conflict merged?

Reply:
Yes. I used vscode. I probably missed it.